notificationsalerts/historical-adherence-query.java

```diff
@@ -44,6 +44,8 @@ public static void main(String[] args) throws IOException, ApiException, Interru
         UsersApi usersApi = new UsersApi();
         UserMe me = usersApi.getUsersMe(Collections.emptyList(), null);
         String myId = me.getId();
+        // When not operating in the context of a user use the oauth client ID instead
+        // final String clientId = System.getenv("GENESYS_CLOUD_CLIENT_ID");
```

Comment:
What is this supposed to convey? The variable `clientId` isn't used anywhere in this file.

Reply:
The intent was to use this `clientId` instead of `myId` when there is no user context. It would be used in the same places `myId` is used, but I could just change `clientId` to `myId` if that makes it clearer.

Reply:
I think using the same variable would make sense so users can change the functionality of the program by changing which line is commented instead of having to refactor the app depending on which one they use.